glyphs2fontir/src/source.rs

```diff
@@ -912,98 +927,61 @@ fn check_pos(
     Ok(())
 }
 
-impl Work<Context, WorkId, Error> for GlyphIrWork {
+impl Work<Context, WorkId, Error> for BracketGlyphIrWork {
     fn id(&self) -> WorkId {
         WorkId::Glyph(self.glyph_name.clone())
     }
 
     fn read_access(&self) -> Access<WorkId> {
+        // We need access to the glyph whence we emerged (bracket glyphs come from non-bracket glyphs)
+        // and it's anchors
         AccessBuilder::new()
             .variant(WorkId::StaticMetadata)
             .variant(WorkId::GlobalMetrics)
+            .specific_instance(WorkId::Glyph(self.parent_glyph_name().clone()))
+            .specific_instance(WorkId::Anchor(self.parent_glyph_name().clone()))
```

Comment:
I was thinking about this earlier and I suspect we don't want the parent anchors? we want to take anchors specifically from the relevant layers, right?

(I acknowledge this is my bug)

Reply:
Can I merge and leave that to you as follow-on? - I'm strapped for time [and irritable it lol]